hastexo/hastexo.py

```diff
@@ -396,7 +396,7 @@ def launch_stack(self, stack, settings, reset=False):
                 provider_name = "default"
             else:
                 try:
-                    provider_name = configured_providers.iterkeys().next()
+                    provider_name = next(iter(configured_providers))
```

Comment:
@arbrandes This is the line that I'm least sure of in the whole commit. Does this _actually_ iterate? (Also, did it before?)

`iter(d)` is a valid shorthand for `iter(d.keys())`, that part I did check.

Reply:
This is just pythonese for "get me the first entry".  The change works just as well, AFAICT.